### PR TITLE
Added blueprint check

### PR DESCRIPTION
**Pull request by dafortin**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [x] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
* **Added** blueprint check

### CODE REVIEW

File: blueprints/ember-frost-sidebar/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
+module.exports = {
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-frost-core', target: '^1.14.3'},
```
Comment (rox163) [inline blueprints/ember-frost-sidebar/index.js]: do we need computed decorators here?

**Comment by rox163** [comment]
:+1:

**Comment by rox163** [comment]
:+1:

**Comment by sglanzer** [comment]
Approved

**Pull request merged**